sdk/translation/ai-translation-document-rest/review/ai-translation-document.api.md

```diff
@@ -212,7 +205,7 @@ export interface FileFormatOutput {
 }
 
 // @public
-export type FileFormatType = "document" | "glossary" | string;
+export type FileFormatType = string;
```

Comment:
Why did we remove the extensible enum for this?

Reply:
Also an emitter change. The old type collapses to `string` anyway so adding the constants to the union doesn't provide any value for intellisense etc. The known values are documented in the tsdoc

Cc @joheredi